main/feed_data.R

```diff
@@ -1,11 +1,17 @@
+library(plyr)
+library(readr)
+
 df <- data.frame(
```

Comment:
this df isn't used anymore. delete if both_t_data completely replaces it? this df seems nice since it's a fairly minimal test, though.

Reply:
removed